packages/p2p-v2/src/pages/advertiser/screens/AdvertiserAdvertsTable/__tests__/AdvertiserAdvertsTable.spec.tsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import { APIProvider, AuthProvider, p2p } from '@deriv/api-v2';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import AdvertiserAdvertsTable from '../AdvertiserAdvertsTable';
+
+let mockApiValues = {
+    isFetching: false,
+    isLoading: true,
+    loadMoreAdverts: jest.fn(),
+};
+
+jest.mock('@deriv-com/ui', () => ({
+    ...jest.requireActual('@deriv-com/ui'),
+    useDevice: jest.fn(() => ({ isMobile: false })),
+}));
+
+jest.mock('@deriv/api-v2', () => ({
+    ...jest.requireActual('@deriv/api-v2'),
+    p2p: {
+        advert: {
+            useGetList: jest.fn(() => mockApiValues),
+        },
+        advertiser: {
+            useGetInfo: jest.fn(() => ({ data: { id: '123' } })),
+        },
+        paymentMethods: {
+            useGet: jest.fn(() => ({ data: [] })),
+        },
+    },
+    // useExchangeRateSubscription: {
```

Comment:
as of now is this being used? if not we can remove it or add a todo if needed later.

Reply:
ah forgot to remove this